templates/web/common/site_layout.tt.html

```diff
@@ -6,6 +6,7 @@
     <meta charset="utf-8">
     <title>[% title %]</title>
     [% meta_description %]
+  	<meta name="apple-itunes-app" content="app-id=588797948">
```

Comment:
we shall put it in a variable like for fb_config, and have a condition of this variable being defined.

This is because product opener is used for multiple projects (eg: openbeautyfacts).

Reply:
oups, didn't see @stephanegigandet comment !

Reply:
Cool, thanks for the info guys